doc/api/debugger.markdown

```diff
@@ -4,9 +4,11 @@
 
 <!-- type=misc -->
 
-V8 comes with an extensive debugger which is accessible out-of-process via a
-simple [TCP protocol][]. Node.js has a built-in client for this debugger. To
-use this, start Node.js with the `debug` argument; a prompt will appear:
+V8 includes a full-featured debugging utility that is accessible out-of-process
+via a simple [TCP-based protocol][]. Node.js provides a built-in client for this
```

Comment:
Technically speaking that's no longer true; the TCP endpoint is provided by node, V8 only provides a programmatic API.